lib/xclarity_client/configuration.rb

```diff
@@ -21,8 +22,15 @@ def initialize(args)
         @auth_type = 'basic_auth'
       end
 
-      $lxca_log.info "XClarityClient::Configuration initialize","Configuration built successfuly"
+      unless @timeout.is_a?(Integer) || @timeout.nil?
```

Comment:
Please, prefer access props through its get methods instead access it directly, once we have `attr_accessor` defined.